arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json

```diff
@@ -555,6 +555,72 @@
         }
       }
     },
+    "/webhdfs/v1/{flushFilePath}": {
```

Comment:
We will have to revisit this again. This is an invalid swagger spec from swagger specification point of view. We may need to correct this by putting all the overloaded paths inside x-ms-paths.